docs-src/0.4/en/reference/component_props.md

```diff
@@ -151,6 +164,10 @@ fn TitleCard(cx: Scope, title: String) -> Element {
 
 > While the new Component is shorter and easier to read, this macro should not be used by library authors since you have less control over Prop documentation.
 
+As we mentioned previoisly in the part about lifetimes and memoization, the update function borrows from the props struct; as the `#[component]` is only a shorthand way of defining a component function and a Props struct in one go, this does not change.
+
+In the `TitleCard` example, the type of `title` is atually `title: &String`, as it is borrowed from the component scope!
```

Comment:
atually -> actually

Reply:
It might be more clear to say something like the type of `title` inside the function is `&String` to make it clear you still need to pass in a type `String` when you create the component